greedy_method/test_*.py

```diff
@@ -0,0 +1,21 @@
+import unittest
```

Comment:
Please change the filename to `test_knapsack.py`

Reply:
Sure thing! Thank you @cclauss !